client/src/components/common/boost/BoostResults.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { BoostResultT } from 'types/boost';
+import { Card, Accordion, Table } from 'react-bootstrap';
+
+export interface BoostResultProps {
+  results: BoostResultT | null;
+}
+
+/**
+ * Displays the Maximum achievable speed and the power plan (the distance and the associated recommended power)
+ *
+ * @param props BoostResultProps
+ * @returns JSX Element
+ */
+export default function BoostResult(props: BoostResultProps) {
+  const { results } = props;
+
+  return (
+    <Card className="mt-4">
+      <Card.Body>
+        <Card.Title className="small">
+          <b>BOOST Output:</b>
+        </Card.Title>
+        <div className="small pb-3">
+          Maximum Speed
+          <span className="float-right pr-4">
+            {results ? `${results.maxSpeed} km/h` : 'N/A'}
```

Comment:
Can you format this to two decimal places? Similarly for the distance and power, but these can probably be rounded to integers.